internal/core/src/query/generated/ExecPlanNodeVisitor.h

```diff
@@ -68,6 +68,31 @@ class ExecPlanNodeVisitor : public PlanNodeVisitor {
         return ret;
     }
 
+    void
+    SetExprCacheOffsets(std::vector<int64_t>&& offsets) {
+        expr_cached_pk_id_offsets_ = std::move(offsets);
+    }
+
+    void
+    AddExprCacheOffset(int64_t offset) {
+        expr_cached_pk_id_offsets_.push_back(offset);
+    }
+
+    const std::vector<int64_t>&
+    GetExprCacheOffsets() {
+        return expr_cached_pk_id_offsets_;
+    }
+
+    void
+    SetExprUsePkIndex(bool use_pk_index) {
+        expr_use_pk_index_ = use_pk_index;
```

Comment:
remove SetExprUsePkIndex

use expr_cached_pk_id_offsets_ == null or expr_cached_pk_id_offsets_.isEmpty()

Reply:
expr_cached_pk_id_offsets_.isEmpty() is not correct because some situation offsets is empty when use cache.
expr_cached_pk_id_offsets == null is not code read friendly